srcpkgs/caps2esc/files/caps2esc.yaml.example

```diff
@@ -0,0 +1,4 @@
+- JOB: "intercept -g $DEVNODE | caps2esc | uinput -d $DEVNODE"
```

Comment:
Doesn't upstream have their own examples? I'd rather not vendor this.

Reply:
It does. I'll remove this example in the next commit.